src/web/index.jsx

```diff
@@ -12,6 +12,7 @@ import SlackNotificationForm from './event-notifications/event-notification-type
 import SlackNotificationSummary from './event-notifications/event-notification-types/SlackNotificationSummary';
 
 import packageJson from '../../package.json';
+import SlackNotificationDetails from "./event-notifications/event-notification-details/SlackNotificationDetails";
```

Comment:
I expect the linter to complain about this line.  Should probably be between `PagerDutyNotificationSummary` and `SlackNotificationForm`